### PR TITLE
squid:CommentedOutCodeLine - Sections of code should not be commented out.

### PR DESCRIPTION
**Pull request by georgekankava**
This pull request is focused on resolving occurrences of Sonar rules
squid:CommentedOutCodeLine - Sections of code should not be commented out.
You can find more information about the issue here:
https://dev.eclipse.org/sonar/rules/show/squid%3ACommentedOutCodeLine
Please let me know if you have any questions.
George Kankava

### CODE REVIEW

File: dashbuilder-webapp/src/main/java/org/dashbuilder/backend/SalesDataSetGenerator.java [hunk 1]
```diff
@@ -134,7 +134,7 @@ private Date buildDate(int month, int year) {
         return c.getTime();
     }
 
-    private Date addDates(Date d, int days) {
+    private static Date addDates(Date d, int days) {
```
Comment (psiroky) [inline dashbuilder-webapp/src/main/java/org/dashbuilder/backend/SalesDataSetGenerator.java]: Why changing the method to be static?

Comment (georgekankava) [inline dashbuilder-webapp/src/main/java/org/dashbuilder/backend/SalesDataSetGenerator.java]: Following sonar rule  squid:S2325 methods that don't access instance data should be static. But it is optional, I can remove this change from this commit if you would like.

Comment (psiroky) [inline dashbuilder-webapp/src/main/java/org/dashbuilder/backend/SalesDataSetGenerator.java]: I'll leave the decision to @dgutierr

Comment (dgutierr) [inline dashbuilder-webapp/src/main/java/org/dashbuilder/backend/SalesDataSetGenerator.java]: Better to leave it as non-static. To make it static private it does not really make any difference in my opinion.

**Comment by georgekankava** [comment]
done :)

**Comment by dgutierr** [comment]
+1 ok to test

**Pull request closed without merging**